tafinder/app/models/term.rb

```diff
@@ -1,6 +1,6 @@
 class Term < ActiveRecord::Base
   has_many :rankings, dependent: :nullify
```

Comment:
We're actually doing that right now. Every relation has a unique ID called id, and then in the application model we verify that student_id and term_id are unique.